tests/vmi_lifecycle_test.go

```diff
@@ -687,28 +716,41 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 				curVMI, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred(), "Should get VMI")
-				Expect(curVMI.Status.NodeName).To(Equal(nodes.Items[0].Name), "Updated VMI name run on the node with matching NFD CPU label")
+				Expect(curVMI.Status.NodeName).To(Equal(node.Name), "Updated VMI name run on the node with matching NFD CPU label")
 
 			})
 
-			It("[test_id:1640]the vmi with cpu.model that cannot match an nfd label on node should not be scheduled", func() {
-
-				nodes, err := virtClient.CoreV1().Nodes().List(metav1.ListOptions{})
-				Expect(err).ToNot(HaveOccurred(), "Should list nodes")
-				Expect(nodes.Items).ToNot(BeEmpty(), "There should be some nodes")
+			It("[test_id:1640]the vmi with cpu.model cpu.features that cannot match nfd labels on a node should not be scheduled", func() {
```

Comment:
here as well, i don't like that this test is being overloaded. It means we can accidently break one part (feature policy forbid for example) but the test will still pass because the cpumodel prevents it from being scheduled.